api/src/wfl/module/aou.clj

```diff
@@ -47,17 +45,31 @@
    :disk_size                        100
    :preemptible_tries                3})
 
-(defn map-aou-environment
-  "Map AOU-ENV to environment for inputs preparation."
-  [aou-env]
-  ({:aou-dev "dev" :aou-prod "prod"} aou-env))
+(def ^:private known-cromwells
+  ["https://cromwell-gotc-auth.gotc-dev.broadinstitute.org"
+   "https://cromwell-aou.gotc-prod.broadinstitute.org"])
 
-(defn env-inputs
-  "Array inputs for ENVIRONMENT that do not depend on the input file.
-  Environment here is just a default, input file may specify override."
-  [environment]
-  {:vault_token_path (get-in env/stuff [environment :vault_token_path])
-   :environment      (map-aou-environment environment)})
+(def ^:private inputs+options
+  [{:environment "dev"
+    :vault_token_path "gs://broad-dsp-gotc-arrays-dev-tokens/arrayswdl.token"}
+   {:environment "prod"
+    :vault_token_path "gs://broad-dsp-gotc-arrays-prod-tokens/arrayswdl.token"}])
+
+;; visible for testing
+(defn cromwell->inputs+options
+  "Map cromwell URL to workflow inputs and options for submitting an AllOfUs Arrays workflow.
+  The returned environment string here is just a default, input file may specify override."
+  [url]
+  ((zipmap known-cromwells inputs+options) (util/de-slashify url)))
+
+(defn ^:private is-known-cromwell-url?
+  [url]
+  (if-let [known-url (->> url
+                          util/de-slashify
+                          ((set known-cromwells)))]
+    known-url
+    (throw (ex-info "Unknown Cromwell URL provided."
+                    {:cromwell url}))))
```

Comment:
Or you could just add both variants to the set.

Reply:
Good idea! Will do.